app/src/main/java/com/example/stretchy/di/RepositoryComponent.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.stretchy.di
+
+import com.example.stretchy.features.executetraining.ui.ExecuteTrainingViewModel
+import com.example.stretchy.features.traininglist.ui.TrainingListViewModel
+import dagger.Component
+
+@Component(modules = [DataBaseModule::class])
+interface RepositoryComponent {
+    fun inject(executeTrainingViewModel: ExecuteTrainingViewModel)
+    fun inject(trainingViewModel: TrainingListViewModel)
```

Comment:
there should be only components per eaither `Application` or `Activity` or `Fragment` or sometimes for a feature, no components per other classes, for now, lets create 2 components `ApplicationComponent` and `ActivityComponent` the second should be dependent on first one